oracle/src/services/RpcUrlsManager.js

```diff
@@ -15,19 +19,22 @@ function RpcUrlsManager(homeUrls, foreignUrls) {
   this.foreignUrls = foreignUrls.split(',')
 }
 
-RpcUrlsManager.prototype.tryEach = async function(chain, f) {
+RpcUrlsManager.prototype.tryEach = async function(chain, f, redundant = false) {
```

Comment:
should it be covered by somehow in the tests?

Reply:
I added tests for `promiseAny` function, similar to the tests we have for `tryEach` util function. I think this should be enough.